lib/user-spec.js

```diff
@@ -0,0 +1,19 @@
+const assert = require('assert');
+const mediator = require('fh-wfm-mediator/lib/mediator');
+const store = require('./userMemoryStore');
+require('./user')(mediator, store);
+
+describe('user topics', function() {
+  describe('auth', function() {
+    it('should read single', function(done) {
+      mediator.request('wfm:user:auth',{
+        username: 'trever',
+        password: '123'
+      }, {uid: 'trever'})
```

Comment:
@witmicko updated this to support `mediator.request`, see this unit test for example.